packages/contracts-bsc/contracts/BscTokenController.sol

```diff
@@ -610,6 +611,7 @@ contract BscTokenController {
      */
     function setCanBurn(address burner, bool canBurn) external onlyRegistryAdminOrOwner {
         token.setCanBurn(burner, canBurn);
+        emit CanBurn(burner, canBurn);
```

Comment:
I noticed that you added an event here in `BscTokenController`. Based on the coding style of these contracts, I believe it would be more appropriate to emit the event from inside `TrueCurrency` than here, for two reasons:
- other `token` composition functions in `BscTokenController` do not emit events, so this would be inconsistent style
- no direct contract storage gets changed for `BscTokenController` due to this function, but contract storage _does_ get changed in `TrueCurrency` (which doesn't currently emit an event for `token.setCanBurn()`).

I would also recommend making this change in a separate PR.

Reply:
will separate it